src/core/tests/visitors/op/glu.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ov_ops/glu.hpp"
+
+#include <gtest/gtest.h>
+
+#include "visitors/visitors.hpp"
+
+using ov::PartialShape;
```

Comment:
Declaration for PartialShape seem to be not used. Is there any reason why GLU is moved to TEST namespaces causing duplication?

Reply:
I consider it as a test detail, so I was following other visitors test file for other op. I don't see specific reason to repeat the `using ov::op::internal::GLU` for each test, updated, `using ov::PartialShape` removed.